test/functional/feature_nulldummy.py

```diff
@@ -77,7 +78,8 @@ def run_test(self):
         cms = self.nodes[0].createmultisig(1, [self.pubkey])
         wms = self.nodes[0].createmultisig(1, [self.pubkey], 'p2sh-segwit')
         self.ms_address = cms["address"]
-        ms_unlock_details = {"scriptPubKey": self.nodes[0].validateaddress(self.ms_address)["scriptPubKey"],
+        spk = address_to_scriptpubkey(self.ms_address).hex()
+        ms_unlock_details = {"scriptPubKey":spk,
```

Comment:
I'm not sure adding `spk` as a variable really helps anything here and it does add an extra line to be reviewed. For a refactor, I'd suggest keeping the diff as small and clear as possible

```suggestion
        ms_unlock_details = {"scriptPubKey": address_to_scriptpubkey(self.ms_address).hex(),
```

Reply:
Thanks.